skrub/_dataframe/_namespace.py

```diff
@@ -97,3 +97,8 @@ def get_df_namespace(*dfs):
                 "Only Pandas or Polars dataframes are currently supported, "
                 f"got {modules=!r}."
             )
+
+
+def skrubns(*dataframes):
```

Comment:
fair enough I can remove it. the gain is we almost never need the second item returned by `get_df_namespace` so I found myself repeatedly typing `df, _ = get_df_namespace` which is ok in an assignment but not possible in an expression which would require writing `get_df_namespace[0]` but then the 0 seems a bit magic

Reply:
Ok, it makes sense. We could instead split the `get_df_namespace` in two functions, returning either the namespace or the skrub namespace, so that you don't have to type `df, _ = ....`
WDYT?